projects/developer/src/app/common/components/queue-load/component.ts

```diff
@@ -19,10 +20,16 @@ export class QueueLoadComponent implements OnInit, OnDestroy, OnChanges {
     @Input() jobTypeIds: any;
     @Input() maintainAspectRatio: boolean;
     @Output() chartLoaded: EventEmitter<UIChart> = new EventEmitter<UIChart>();
+    themeSubscription: any;
```

Comment:
I would prefer to see `this.subscription: any` change to `this.subscriptions: Subscription[] = []`, which would allow you to subscribe like this `this.subscriptions.push(this.themeService.subscribe( ... ))` and the `ngOnDestroy()` would just be `this.subscriptions.forEach(s => s.unsubscribe())`. I really don't care if you fix this though, just for your future reference.